src/network/networktool.cpp

```diff
@@ -12,39 +12,39 @@
 
 PING_LOGGING_CATEGORY(NETWORKTOOL, "ping.networktool")
 
-
-QString NetworkTool::_gitUserRepo = "bluerobotics/ping-viewer";
-QString NetworkTool::_gitUserRepoFirmware = "bluerobotics/ping-firmware";
-
+namespace {
+    QString gitUserRepo = QStringLiteral("bluerobotics/ping-viewer");
+    QString gitUserRepoFirmware = QStringLiteral("bluerobotics/ping-firmware");
+    const QString gitReleases = QStringLiteral("https://api.github.com/repos/%1/releases");
+    const QString repoContents = QStringLiteral("https://api.github.com/repos/%1/contents/%2");
+    const QString projectTag = QStringLiteral(GIT_TAG);
+    const QString userRepoUrl = QStringLiteral(GIT_URL);
+}
```

Comment:
Add a space between namespace and class constructor, also the commit does not describes what is happening here and why this namespace is necessary.